sdk/core/core-amqp/src/auth/sas.ts

```diff
@@ -60,9 +57,7 @@ export class SasTokenProvider implements TokenProvider {
     this.tokenValidTimeInSeconds = tokenValidTimeInSeconds || 3600;
     this.tokenRenewalMarginInSeconds = tokenRenewalMarginInSeconds || 900;
     if (this.tokenValidTimeInSeconds <= this.tokenRenewalMarginInSeconds) {
-      throw new Error(
-        "tokenRenewalMarginInSeconds must be less than tokenValidTimeInSeconds"
-      );
+      throw new Error("tokenRenewalMarginInSeconds must be less than tokenValidTimeInSeconds");
```

Comment:
We don't have a use case currently where we want `tokenValidTimeInSeconds` and `tokenRenewalMarginInSeconds` to be customizable by the user. So let's remove them from the constructor